### PR TITLE
openapi: improve information about device and tags routes

### PR DESCRIPTION
**Pull request by henrybarreto**
I would like a little help to solve the To-dos commented on the code

### CODE REVIEW

File: openapi/spec/resources/devices/auth.yaml [hunk 1]
```diff
@@ -15,47 +15,110 @@ post:
...
+                  type: string
+                  example: latest
+                arch:
+                  description: Device's OS arch
+                  type: string
+                  # TODO: add more compatible arches.
+                  enum:
+                    - x86_64
+                    - amd64
+                    - arm64
```
Comment (otavio) [inline openapi/spec/resources/devices/auth.yaml:51]: I'd not validate it, at this moment. We support anything which runs Go.

**Review by otavio — DISMISSED** [review]
Auth and device model are quite close ... is this intended?

File: openapi/spec/resources/devices/auth.yaml (hunk 1 above)
Comment (henrybarreto) [reply openapi/spec/resources/devices/auth.yaml:51]: Perfect!

**Comment by henrybarreto** [comment]
> Auth and device model are quite close ... is this intended?

Yes. That auth is the device's auth used by agent.

**Pull request closed without merging**